challenges/pt-br/sre-challenge.md

```diff
@@ -1,27 +1,23 @@
 # Hash Teste SRE
 
-Antes de começar, leia a nossa [página sobre cultura](https://tech-culture.hash.com.br/) para entender um pouco sobre o que nós priorizamos no desenvolvimento e faça o seu melhor, pois iremos avaliar o teste como se fosse seu melhor esforço ;)
+Olá candidato! Antes de começar, leia a nossa [página sobre cultura](https://tech-culture.hash.com.br/) para entender um pouco sobre o que nós priorizamos no desenvolvimento e faça o seu melhor, pois iremos avaliar o teste como se fosse seu melhor esforço ;)
 
-O teste de SRE é dividido em **arquitetura** e **desenvolvimento** e complementa o [desafio de back end](https://github.com/hashlab/hiring/blob/master/challenges/pt-br/back-challenge.md).
-
-> Se você estiver fazendo parte do processo seletivo iremos enviar um arquivo zip com o desafio de backend
+O teste consiste em dois desafios. O primeiro deles explora seu entendimento sobre arquitetura de microserviços e o segundo suas habilidades de automação.
 
 ## Arquitetura
 
-- Os dois cenários a seguir são baseados no [desafio de backend](https://github.com/hashlab/hiring/blob/master/challenges/pt-br/back-challenge.md), leia-os antes de prosseguir.
+#### O resultado deve conter um documento de texto (pdf, markdown, git, etc) contendo a arquitetura proposta e documentação relevante (se houver).
+- **Opcional.** Caso queira demonstrar a arquitetura em código de forma reprodutível também seria bem legal. (ex. Terraform e/ou outros)
```

Comment:
@ponceps the changes I am suggesting are related to the following patterns/references, what do you think?

- https://www.w3.org/WAI/tutorials/page-structure/headings/
- https://cirosantilli.com/markdown-style-guide/#punctuation-at-the-end-of-headers

```suggestion
### O resultado deve conter um documento de texto (pdf, markdown, git, etc) contendo a arquitetura proposta e documentação relevante (se houver)

- **Opcional.** Caso queira demonstrar a arquitetura em código de forma reprodutível também seria bem legal. (ex. Terraform e/ou outros)
```